ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/TrailingCommaOnDeclarationSiteRule.kt

```diff
@@ -364,6 +365,8 @@ public class TrailingCommaOnDeclarationSiteRule :
             val lastChild = element.collectDescendantsOfType<KtCollectionLiteralExpression>().last()
             containsLineBreakInLeavesRange(lastChild.rightBracket!!, element.rightParenthesis!!)
         }
+        element is KtValueArgumentList && element.arguments.isEmpty() -> false
```

Comment:
This line is redundant as the value argument list is handled by the call site rule